cmd/main.go

```diff
@@ -59,3 +73,53 @@ func SupersimMain(ctx *cli.Context, closeApp context.CancelCauseFunc) (cliapp.Li
 
 	return s, nil
 }
+
+func isMinAnvilInstalled(log log.Logger) (bool, error) {
+	cmd := exec.Command("anvil", "--version")
+	var out bytes.Buffer
+	cmd.Stdout = &out
+	err := cmd.Run()
+	if err != nil {
+		return false, err
+	}
+
+	output := out.String()
+
+	versionRegex := regexp.MustCompile(`\d+\.\d+\.\d+`)
+	version := versionRegex.FindString(output)
+
+	if version == "" {
+		return false, fmt.Errorf("failed to parse anvil version from output")
+	}
+
+	log.Info(fmt.Sprintf("anvil version: %s installed", version))
+
+	if isVersionGreaterOrEqual(version, minAnvilVersion) {
+		return true, nil
+	}
+
+	return false, nil
+}
+
+// Compares two semantic versions.
+func isVersionGreaterOrEqual(version, minVersion string) bool {
```

Comment:
nit: I think this package has the functionality you're looking for https://pkg.go.dev/golang.org/x/mod/semver
```
func Compare(v, w string)
```

Reply:
prob ignore this if we're not going with semver